dev/ipynb_to_gallery.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
@registerrier and @adonath, I have just realised that all scripts in the dev repo have not as first line the reference to our licence. Should we add it in a separate PR?